docs/how-to/dss.rst

```diff
@@ -0,0 +1,181 @@
+
+Get DSS status
+--------------
+
+You can check the DSS status through ``dss status``. 
+This command provides a quick way to check the status of your DSS environment, including the MLflow status and whether a GPU is detected in your environment.
+
+.. code-block:: bash
+
+    dss status
+
+If you already have a DSS environment running and no GPU available, the expected output is:
+
+.. code-block:: none
+
+    [INFO] MLflow deployment: Ready
+    [INFO] MLflow URL: http://10.152.183.68:5000
+    [INFO] GPU acceleration: Disabled
+
+List DSS commands
+-----------------
+
+You can list the available commands to interact with DSS.
+
+**List the commands**:
+
+    To get the list of available commands for DSS, run ``dss`` with the ``--help`` option:
+
+    .. code-block:: bash
+
+        dss --help
+
+    You should expect an output like this:
+
+    .. code-block:: none
+
+        Usage: dss [OPTIONS] COMMAND [ARGS]...
+
+        Command line interface for managing the DSS application.
+
+        Options:
+        --help  Show this message and exit.
+
+        Commands:
+        create      Create a Jupyter notebook in DSS and connect it to MLflow.
+        initialize  Initialize DSS on the given Kubernetes cluster.
+        list        Lists all created notebooks in the DSS environment.
+        logs        Prints the logs for the specified notebook or DSS component.
+        purge       Removes all notebooks and DSS components.
+        remove      Remove a Jupter Notebook in DSS with the name NAME.
+        start       Starts a stopped notebook in the DSS environment.
+        status      Checks the status of key components within the DSS...
+        stop        Stops a running notebook in the DSS environment.
+
+    
+**Get details about a specific command**:
+
+    To see the usage and options of a DSS command, run ``dss <command>`` with the ``--help`` option.
+    For example:
+
+    .. code-block:: bash
+
+        dss logs --help
+
+    You should expect an output like this:
+
+    .. code-block:: none
+
+        Usage: dss logs [OPTIONS] [NOTEBOOK_NAME]
+
+        Prints the logs for the specified notebook or DSS component.
+
+        Examples:
+            dss logs my-notebook
+            dss logs --mlflow
+            dss logs --all
+
+        Options:
+        --kubeconfig TEXT  Path to a Kubernetes config file. Defaults to the value
+                            of the KUBECONFIG environment variable, else to
+                            './kubeconfig'.
+        --all              Print the logs for all notebooks and MLflow.
+        --mlflow           Print the logs for the MLflow deployment.
+        --help             Show this message and exit.
+
+See also
+--------
+
+To learn how to manage your Jupyter Notebooks, check :ref:`manage_notebooks`. 
+If you are interested in managing MLflow within your DSS environment, see :ref:`manage_MLflow`.
```

Comment:
Should we have this as a list instead? My personal preference is that it's a bit more readable, but don't have a hard opinion.

Was looking at the `Next Steps` from MicroK8s as a nice example IMO
https://microk8s.io/docs/getting-started